website/docs/guides/0220-usage-testing-disks.md

```diff
@@ -5,6 +5,7 @@ VC automates disk formating and run storage workloads on the target systems. We
 1. On OS Disk, only run on the OS partition.
 2. If other disks have more than 1 partition, run on all concurrently that passes the filters (exclude /, /mnt, /boot.efi)
 3. On Linux, only run on four prefix: `/dev/hd`, `/dev/sd`, `/dev/nvme`, `/dev/xvd`.
+3. On Windows, only disks with status: online & read only disabled will be used.
```

Comment:
I would change this to 
4. On Windows, disks with status set to offline and read-only will be filtered out. 

But feel free to check in, I can edit the doc